packages/nextjs-contentful-renderer/src/utils/getApolloClient/getApolloClient.ts

```diff
@@ -0,0 +1,16 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { createContentfulLink } from '@cmpsr/contentful-core';
+
+export const getApolloClient = (
```

Comment:
Should we make an object with these 4 parameters?

Reply:
done!